test/test-deb-releases.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
```

Comment:
Having this test is a great idea btw! We should integrate some form of it! Maybe for the test itself we could switch it to using systemd-nspawn?

Reply:
What would be the best way doing this with nspawn? From what I've seen it will not be as trivial to do this with nspawn as with docker containers.

Reply:
I'm not 100% sure, but there are many ways to pass an image into nspawn, via debootstrap, a filesystem image, etc...

I'd recommend doing the main part of building things properly and we'll add the test in step 2?

Reply:
Running an image in nspawn is not really the problem I guess. Getting the image with the desired content on is a bit more complicated, that is why docker is so popular.

Doing the tests first would be preferred for me. Because it acts as the proof things are working.